src/App.js

```diff
@@ -1,9 +1,10 @@
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
-      <p>Hello, World!</p>
+      <Calculator />
     </div>
```

Comment:
- [ ] Good job so far 👏 It is important to note that the `App.js` component should be class-based. Please update it 👍

Reply:
## Hi @BertrandConxy! 👋 

Thanks for your time reviewing my project. I really appreciate this.

I've used the `App` component just as React created this one and it works OK, because this component is stateless, so eslint pushes to use it in this way.

I know the requirement ties to use class-type components only, but eslint rules push to use the function for stateless components (check [this](https://itnext.io/react-component-class-vs-stateless-component-e3797c7d23ab)). In my project, the state is linked to the component `calculator` (by the moment 😃 ).

https://github.com/Camilovelag/math-magicians/blob/18d78b49f3e1c1975b4f8c2d8d9e603c092ba0b1/src/components/calculator.js#L1-L17


I've tried changing this one as a class, but eslint throws the following error (see screenshot).

![image](https://user-images.githubusercontent.com/106117541/197885849-8c296ccc-34cc-4a32-85c7-bfee3cdb4f93.png)

At least for me, the requirement makes reference to new components, because it was mentioned before using the component `calculator` inside the _main component_ (App.js).

Could you clarify your review?

Thank you! and happy coding 👏

Reply:
[REQUIRED] You are making reference to the calculator component. The reviewer requested changes in the App component. So instead of `function App` you should have `class App` just like your calculator. The function-based component also works and it is the most commonly used nowadays but the instruction for this specific task mentions that you should use a class-based component at this stage.